gibberish_server/app.js

```diff
@@ -3,12 +3,27 @@ const logger = require('morgan');
 const express = require('express');
 const bodyParser = require('body-parser');
 const http = require('http');
+const mongoose = require('mongoose')
+require('dotenv').config()
+
+const questionbank = require("./mongo_models/question_bank");
 
 const app = express();
 const server = http.createServer(app);
 const PORT = 4000;
 
-server.listen(PORT)
+// connect to mongo database
+mongoose.connect(process.env.MONGODB_URL, { useUnifiedTopology: true, useNewUrlParser: true, useCreateIndex: true });
+
+const connection = mongoose.connection;
+
+connection.once("open", function () {
+    console.log("MongoDB database connection established successfully");
```

Comment:
we have imported morgan logger right? I think its better to use that one instead

Reply:
I don't think morgan logger is used as a replacement for console.log. It basically logs the requests made by the client to our server. https://coralogix.com/log-analytics-blog/morgan-npm-logger-the-complete-guide/

Reply:
oh so it is only used to log the req and res?